### PR TITLE
Pattern tests

### PR DESCRIPTION
**Pull request by bandvov**
refactored and improved tests for patterns

### CODE REVIEW

File: src/redux/auth/auth.sagas.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ function* handleAdminCheckByToken() {
     yield put(setAuth(true));
     yield put(setAuthLoading(false));
   } catch (error) {
+    console.log(error);
```
Comment (TorskyM) [inline src/redux/auth/auth.sagas.js:50]: Remove `console.log`

File: src/components/pattern-form/pattern-form.js [hunk 2]
```diff
@@ -44,7 +44,9 @@ const PatternForm = ({ pattern, id }) => {
   } = usePatternHandlers();
   const languageTabs =
     languages.length > 0
-      ? languages.map((lang, index) => <Tab label={lang} key={lang} />)
+      ? languages.map((lang, index) => (
```
Comment (TorskyM) [inline src/components/pattern-form/pattern-form.js:47]: You do not use the `index` argument, there is no need to provide it

File: src/redux/auth/auth.sagas.js (hunk 1 above)
Comment (bandvov) [reply src/redux/auth/auth.sagas.js:50]: fixed

File: src/components/pattern-form/pattern-form.js (hunk 2 above)
Comment (bandvov) [reply src/components/pattern-form/pattern-form.js:47]: fixed

**Review by TorskyM — APPROVED** [review]

**Review by ecl1pseo — APPROVED** [review]

**Pull request merged**